kubernetes/token.go

```diff
@@ -20,6 +27,22 @@ func GetKialiToken() (string, error) {
 			}
 			KialiToken = string(token)
 		}
+		tokenRead = time.Now()
 	}
 	return KialiToken, nil
 }
+
+// Check if token expired based on the k configuration
+func IsTokenExpired() bool {
+
+	if tokenExpireDuration == 0 {
+		kConfig := kialiConfig.Get()
+		tokenExpireDuration = time.Duration(kConfig.KubernetesConfig.TokenExpireDuration) * time.Second
+	}
+
+	if time.Since(tokenRead) > tokenExpireDuration {
+		return true
+	} else {
+		return false
+	}
+}
```

Comment:
nit: `IsTokenExpired` is a bit of a misnomer since this is just checking how long it's been since the token was last read directly from disk rather than returning the cached token. This doesn't need to be exported by this package and I'd even say you don't need to add another config option to adjust this interval. The config option here currently adjusts how often we read the token directly from disk vs. returning the cached option and users shouldn't need to adjust that beyond the default 60 seconds. You can set `var tokenExpireDuration = time.Second * 60` and remove the config option. This also saves the trouble of having to make operator/helm changes.

Reply:
I have renamed the function name. 

Note this config option is also used in cache.go to indicate the timer periodicity. 
Do we want to change to a fixed value? The read token periodicity is a recommendation, but not sure if that could change? 
I think it could be more flexible as a configuration option. But if no required, the issue https://github.com/kiali/kiali-operator/pull/563 can be closed.

Reply:
I don't see the need for users to configure it but if we're going to keep the config option I'd suggest renaming it. `TokenExpireDuration` to me makes it sound like we will expire the token in 60 seconds.

Reply:
I have finally removed the config item. 
I think it would make sense if this is an expiration time, but less if this is a timer.